src/search.html

```diff
@@ -4,6 +4,7 @@
 	<input
 		type="search"
 		ng-model="param"
+		ng-keyup="change($event)"
```

Comment:
What about the paste event? Consider using a combination of ngChange and ngPaste.